app/utils/fhir-paged-remote-array.js

```diff
@@ -4,9 +4,8 @@ import computed from 'ember-computed';
 export default PagedRemoteArray.extend({
   page: 1,
   totalPages: 0,
-
   sortBy: null,
-  sortDescending: null,
+  sortDesc: false,
```

Comment:
Why was this changed? Line 17 of this file still reads from `sortDescending` which you removed here.